rmw_connext_cpp/src/rmw_init.cpp

```diff
@@ -139,6 +139,8 @@ rmw_init(const rmw_init_options_t * options, rmw_context_t * context)
 
   context->instance_id = options->instance_id;
   context->implementation_identifier = rti_connext_identifier;
+  context->actual_domain_id =
+    RMW_DEFAULT_DOMAIN_ID != options->domain_id ? options->domain_id : 0u;
```

Comment:
@ivanpauno [this line](https://github.com/ros2/rmw_connext/blob/da982d173cdc4d9a4f7ee00683bfefaf743816fd/rmw_connext_shared_cpp/src/node.cpp#L313) has to change too.

Reply:
See https://github.com/ros2/rmw_connext/pull/443/commits/c6ac3bdd4103d73ab8ac8153a30cd8a3ca9d2875.